codegens/dart-dio/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@postman/codegen-dart-dio",
+  "version": "0.0.1",
+  "description": "Converts Postman SDK Requests to Dart Dio code snippet",
+  "main": "index.js",
+  "com_postman_plugin": {
+    "type": "code_generator",
+    "lang": "Dart",
+    "variant": "dio",
+    "syntax_mode": "dart"
+  },
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "node npm/test.js",
+    "test-lint": "node npm/test-lint.js",
+    "test-unit": "node npm/test-unit.js",
+    "test-newman": "node npm/test-newman.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "author": "Postman Labs <help@getpostman.com>",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/postmanlabs/code-generators/tree/master/codegens/dart-dio",
+  "dependencies": {
+    "lodash": "4.17.21"
+  },
+  "devDependencies": {},
+  "engines": {
+    "node": ">=8"
```

Comment:
Let's keep this Node 12. We've updated the main library's min version to be Node 12.